src/builtin.cpp

```diff
@@ -88,7 +88,8 @@ bool builtin_data_t::operator<(const builtin_data_t *other) const {
 ///
 int builtin_count_args(const wchar_t *const *argv) {
     int argc;
-    for (argc = 1; argv[argc] != NULL; argc++);
+    for (argc = 1; argv[argc] != NULL; argc++)
+        ;
```

Comment:
Why does clang-format do this? Is that really the default behavior or have we messed up some options to it? It should create braces for us or just leave it alone.